compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/diagnostics/FirDefaultErrorMessages.kt

```diff
@@ -208,6 +209,7 @@ class FirDefaultErrorMessages : DefaultErrorMessages.Extension {
             map.put(VARIABLE_EXPECTED, "Variable expected")
             map.put(RETURN_NOT_ALLOWED, "'return' is not allowed here")
             map.put(DELEGATION_IN_INTERFACE, "Interfaces cannot use delegation")
+            map.put(NESTED_CLASS_NOT_ALLOWED, "{0} is not allowed here", NULLABLE_STRING)
```

Comment:
Are we going to pass nullable string? Why not `TO_STRING`?

Reply:
Oh, I guess you wanted to nail down to `String` type at least. Either should work, then. `TO_STRING` will weight more on nullability. :)

Reply:
Yeah I wasn't sure what to use. I wish there is an additional `STRING` option :S